From 190374a70659441e74ebc5c8b201c4614059a0c3 Mon Sep 17 00:00:00 2001
From: Joaquin Heredia <joakonat@gmail.com>
Date: Fri, 5 Apr 2013 21:48:30 -0300
Subject: [PATCH 1/6] Update README.md

---
 README.md |    1 +
 1 file changed, 1 insertion(+)

diff --git a/README.md b/README.md
index 4580856..a6ae6dc 100644
--- a/README.md
+++ b/README.md
@@ -1 +1,2 @@
 Alan Gonzalez DNI: 37177303 (marioalangonzalez@gmail.com)
+Joaquin Heredia DNI: 37127698 (joaquin.heredia93@gmail.com)
-- 
1.7.10.4

