From ce0d12dd448a93a794ffd935425411f134c0698e Mon Sep 17 00:00:00 2001
From: Joaquin Heredia <joakonat@gmail.com>
Date: Tue, 16 Apr 2013 16:43:58 -0300
Subject: [PATCH 3/6] Primera parte del SRS, falta fusionar.

---
 SRS  |   86 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 86 insertions(+)
 create mode 100644 SRS 

diff --git a/SRS  b/SRS 
new file mode 100644
index 0000000..90a7273
--- /dev/null
+++ b/SRS 	
@@ -0,0 +1,86 @@
+Universidad Nacional de Río Cuarto.
+Facultad de Ciencias Exactas, Físico-Químicas y Naturales.
+Departamento de Computación.
+Análisis y Diseño de Sistemas (3303).
+
+
+
+
+Especificación de Requisitos de Software
+(SRS)
+
+
+Proyecto:
+Revista Inmobiliaria
+
+
+Integrantes:
+Alan Gonzalez
+Joaquín Heredia
+Santiago Lapiana
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+Índice
+
+
+Registro de Cambios
+1.  Introducción
+2.	Descripción
+3.	Requerimientos Específicos
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+1 Introducción
+
+En este documento se busca definir las especificaciones funcionales, no funcionales y del sistema necesarias para la implementación de una aplicación que permitirá la administración de un sistema para http://Inmobiliariarevista.com.ar  con el fin de permitir el manejo de un registro de usuarios que publiquen avisos, permitir la búsqueda de inmuebles bajo ciertos criterios a definir. Esta aplicación será utilizada por empleados administrativos mediante una interfaz de usuario (GUI).
+
+
+
+
+
+
-- 
1.7.10.4

